docs/docs/40-operator-guide/35-cluster-configuration.md

```diff
@@ -216,3 +216,37 @@ A webhook receiver's only job is to extract a repository URL from the webhook
 request's payload, query for all `Warehouse` resources across all Projects
 having subscriptions to that repository, and request each to execute their
 artifact discovery process.
+
+## Cluster Message Channels
+
+<span class="tag professional"></span>
+<span class="tag beta"></span>
+
+`ClusterMessageChannel` resources are cluster-scoped analogs to
+[`MessageChannel`](../50-user-guide/20-how-to-guides/20-working-with-projects.md#message-channels).
+Their `spec` fields are identical, but `ClusterMessageChannel`s are available to every Project in
+the cluster without needing to create a Project-scoped `MessageChannel` resource. The other
+difference is that `ClusterMessageChannel`s must reference `Secret`s in the designated cluster
+secrets namespace. By default, that namespace is `kargo-cluster-secrets`, but can be changed by the
+operator at the time of installation. (Refer to the [Kargo Helm Chart's
+README.md](https://github.com/akuity/kargo/tree/main/charts/kargo)). For example, here is a
+`ClusterMessageChannel` that sends messages to a Slack channel:
+
+```yaml
+apiVersion: ee.kargo.akuity.io/v1alpha1
+kind: ClusterMessageChannel
+metadata:
+  name: slack
+spec:
+  # A reference to a Secret containing the Slack token. This is required for Slack. The Secret must
+  # contain the following key:
+  # - `apiKey`: The Slack token with permissions to post messages to the desired channel
+  secretRef:
+    # The `namespace` field is ignored for `ClusterMessageChannel` as it is only allowed to reference
+    # Secrets in the cluster-secret-namespace
+    name: slack-token
+  # Configuration specific to Slack
+  slack:
+    # The channel ID to send messages to. This field is required
+    channelID: C1234567890
+```
```

Comment:
Nit: This section, appropriately, seems light, I think, with the assumption that the docs for the Project-scoped analog to this will cover all this in more detail. Do we need to be explicit about referring to those docs for further details and examples? Or do you think it's implied well enough? I could go either way, but thought it was worth asking.

Reply:
Sorry... to clarify... I saw the link above. I just hadn't completely gotten "this section is abbreviated because you should read the other section" from it.